governance/roles.md

```diff
@@ -92,23 +93,38 @@ TLs are assigned by CNCF Technical Oversight Committee
 
 ## Role of project leads
 
-Project Leads will lead larger streams of work that require sustained
-effort and coordination.  In some cases, these projects remain ongoing.
+Project Leads lead specific projects where a deliverable has been defined 
+within the issue.
 
 Project Leads are nominated and approved by the following process:
-  1. Project Lead actively participates in the group, making a proposal or
-  volunteering to take on a project that has been prioritized by the group
+  1. Project Lead actively participates in the group, initiates the proposal or
+  volunteers to take on a project that has been prioritized by the group
   1. A Chair or TL nominates a candidate.
-  1. The nomination is communicated via a pull request annotating the list of members in the 
-  [SIG README](/README.md) with a link to the issue tracking the project. 
+  1. The nomination is communicated on the issue tracking the project 
   The nomination is typically open for a week (but may be shorter with LGTM of at least two Chairs).
-  1. Members are encouraged to review the tracking issue and work together
-  to ensure that the Project Lead is set up for success or suggest alternatives.
+  1. The issue is updated with the assignment of the project lead as "assignee" alongside the SIG Leadership member.
+  1. Project Leads will be given the OWNER role of the directory or sub-directory for their team's activities in order to empower merge/approve.
```

Comment:
I think we need to call out on-going projects.  From past experience, I think it is rare that we create an on-going project and only in that case will there be a directory where someone has OWNER role, right?

Reply:
```suggestion
  1. Project Leads will be given the OWNER role of the directory or sub-directory for their team's activities in order to empower merge/approve.
  
  ### On-going projects
  
  On the occasion where a project becomes long term, Project Leads are also given the OWNER role of the directory or sub-directory along with 1 other nominated individual to ensure continuity in review and ongoing support of the project.
```